bots/sg_single_rule_delete.py

```diff
@@ -7,8 +7,8 @@
 Sample GSL: SecurityGroup should not have inboundRules with [scope = '0.0.0.0/0' and port<=22 and portTo>=22]
 
 Conditions and caveats: Deleting a single rule on a security group can be difficult because the problematic port can be nested within a wider range of ports. If SSH is open because a SG has all of TCP open, do you want to delete the whole rule or would you break up the SG into the same scope but port 0-21 and a second rule for 23-end of TCP port range?
-Currently the way this is being addressed is using the 'split' parameter. If it's set as false, CloudBots will only look for the specific port in question. If it's nested within a larger port scope, it'll be skipped. 
-If you set split to true, then the whole rule that the problematic port is nested in will be removed and 2 split rules will be added in its place (ex: if port 1-30 is open and you want to remove SSH, the new rules will be for port 1-21 and port 23-30). 
+Currently the way this is being addressed is using the 'split' parameter. If it's set as false, CloudBots will only look for the specific port in question. If it's nested within a larger port scope, it'll be skipped.
```

Comment:
Pay attention to https://github.com/dome9/cloud-bots/pull/39
Those changes are also there, probably some git issue so remove those changes from there to avoid conflicts

Reply:
#39 can be ignored, now only this branch needs to be considered 
I created a new pull request for s3 public access ( now I know how to move between branches better after our session )